src/Microsoft.Framework.Runtime.Interfaces/ILibraryInformation.cs

```diff
@@ -6,18 +6,39 @@
 
 namespace Microsoft.Framework.Runtime
 {
+    /// <summary>
+    /// Exposes information about a library which can be an assembly, project, or a package.
+    /// </summary>
     public interface ILibraryInformation
     {
+        /// <summary>
+        /// Gets the name of the library.
+        /// </summary>
         string Name { get; }
 
+        /// <summary>
+        ///  Gets the version of the library.
+        /// </summary>
         string Version { get; }
 
+        /// <summary>
+        /// Gets the path to the library. For projects, this is a project.json file.
+        /// </summary>
         string Path { get; }
 
+        /// <summary>
+        /// Gets the type of library. Common values are Project, Package, and Assembly.
+        /// </summary>
         string Type { get; }
 
+        /// <summary>
+        /// Gets the list of dependencies for the library.
```

Comment:
Since the return dependencies are strings it would useful to document what these strings are (paths, assembly names etc. - I don't know what they are btw.)